test/integration/transaction_test.rb

```diff
@@ -102,9 +108,9 @@ class TransactionTest < ActionDispatch::IntegrationTest
       visit "/foo"
```

Comment:
This doesn't seem to test the statement in the context any more.

Reply:
`assert_has_button_component` [performs an equality test on all button attributes](https://github.com/alphagov/frontend/pull/1402/files/53c9bb105bd37cc57e21d8185dceb4f78b3d0c25#diff-fe8c1aa14e9f516646c2efbae788a973R13) (after extracting `:text`) so this does assert attributes passed to the button do not contain data tracking attrs as in the above test.